keyboards/order/info.json

```
@@ -0,0 +1,17 @@
+{
+    "keyboard_name": "Order",
+    "url": "https://github.com/FrancTheCat/Order",
+    "maintainer": "Carlos Rödiger",
+    "width": 15,
+    "height": 5,
+    "layouts": {
+        "LAYOUT_ortho_15x5": {
```

QMK custom for this is `rows`x`columns`.

```suggestion
        "LAYOUT_ortho_5x15": {
```